public/app/app.component.ts

```diff
@@ -1,7 +1,32 @@
-import {Component} from 'angular2/core';
+import {Component, OnInit} from 'angular2/core';
+import {Http, Response} from 'angular2/http';
+import {User}              from './services/user';
+import {UserService}       from './services/user-service';
+
 
```

Comment:
This is mostly boiler plate copied from angular2 tutorials. I don't quite understand everything that is going on here yet. Mainly the subscribe function and its relation to the UserService. Might need to rename UserService to UserResource.